src/main/menu.js

```diff
@@ -42,7 +42,11 @@ function setWindowFocus (flag) {
 // Disallow opening more screens on top of the current one.
 function setAllowNav (flag) {
   getMenuItem('Preferences').enabled = flag
-  getMenuItem('Create New Torrent...').enabled = flag
+  getMenuItem(
+    process.platform === 'darwin'
+      ? 'Create New Torrent...'
+      : 'Create New Torrent from Folder...'
+  ).enabled = flag
   var item = getMenuItem('Create New Torrent from File...')
   if (item) item.enabled = flag
```

Comment:
Nice catch!
I have a suggestion for making it a little more readable, but I don't mind keeping it like this.
```javascript
if (process.platform === 'darwin') {
    getMenuItem('Create New Torrent...').enabled = flag
} else {
    getMenuItem('Create New Torrent from Folder...').enabled = flag
    getMenuItem('Create New Torrent from File...').enabled = flag
}
```